README.md

```diff
@@ -85,25 +85,25 @@ the following system's environment viaribles:
 
 ##### General configuration
 
-* `FENNEC_LOGLEVEL` - `debug`/`info`/`warn`/`error` - Log level of the application. `info` is the default one
-* `FENNEC_UDP_ENABLED` - `true`/`false` - Enable or disable UDP STUN/TURN interface. Enabled by default
-* `FENNEC_TCP_ENABLED` - `true`/`false` - *Not yet supported* - Enable or disable TCP STUN/TURN interface. Disabled by default.
-* `FENNEC_STUN_SECRET` - Secret that STUN/TURN clients have to use to authorize with the server
+* `MONGOOSEIM_LOGLEVEL` - `debug`/`info`/`warn`/`error` - Log level of the application. `info` is the default one
```

Comment:
All these environment variables should start with `MONGOOSEICE_`, not `MONGOOSEIM_`